MusicApp.SongService.Web/Controllers/SongsController.cs

```diff
@@ -84,4 +86,22 @@ public async Task<IActionResult> DeleteSong(Guid id, CancellationToken cancellat
 
         return NoContent();
     }
+
+    [HttpPatch("{id:guid}")]
+    public async Task<IActionResult> LikeSong(Guid id, CancellationToken cancellationToken)
+    {
+        var likeCommand = new LikeSongCommand(id);
+        await _mediator.Send(likeCommand, cancellationToken);
+
+        return Ok();
+    }
+
+    [HttpGet("/api/songs/favorite")]
```

Comment:
follow one style of route naming

Reply:
which path should i use here?

Reply:
now you have such route as api/songs/api/songs/favorite to this endpoint. remove the duplication and leave favorite in HttpGet

Reply:
It works because when i start the path with '/' it ignores the route in the attribute above the controller.
If I just wrote just "favorite" it would consider it as a call to the "/api/songs/{id}" endpoint with the wrong id.

Reply:
Sorry my bad. Putting just "Favorite" really works)